cis/system/ksp-audit-cis-mysql-3-10.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cis-mysql-3-10
+  namespace: default
+spec:
+  tags: ["MySQL", "CIS", "rule-3.10", "v8", "v7", "Control-3.3", "Control-14.6"]
+  message: "Notification! /etc/mysql/my.cnf has been accessed! keyrings may have been modified."
+  selector:
+    matchLabels:
+      app: mysql 
+  file:
+    severity: 1
+    matchPaths:
+    - path: /etc/mysql/my.cnf
```

Comment:
Please change the path to possible  <keyring file> and add owneronly flag @yasin-cs-ko-ak 
cc: @syeddhadi @raviknox @salman-accuknox

Reply:
@yasin-cs-ko-ak sorry, the part where I mentioned keyring got omitted in the comment. Please refer to https://dev.mysql.com/doc/refman/8.0/en/keyring-system-variables.html and find the keyring path and protect it with the owneronly flag. also please remove my.conf file from the path as well as message

Reply:
keyring paths and directory has been added. thanks for reviewing!